### PR TITLE
fix: Remove unused files from project

### PR DESCRIPTION
**Pull request by mateusoliveira43**
## Describe the behavior changes introduced in this PR

Remove unused files from the project.

To help me, I have run
```
grep -Iinr 'demo' . --exclude-dir={.venv,htmlcov,.pytest_cache,.mypy_cache}
grep -Iinr 'labs' . --exclude-dir={.venv,htmlcov,.pytest_cache,.mypy_cache}
grep -Iinr 'samples' . --exclude-dir={.venv,htmlcov,.pytest_cache,.mypy_cache}
```

## Linked Issues

No.

## Testing Instructions

Check if all unused files where removed (and no used file was removed).

### CODE REVIEW

File: samples/todolist_commit_deploy.md
Comment (mpryc) [inline samples/todolist_commit_deploy.md:1]: Should this be rewritten to match the Operator instead of removal?

Comment (mateusoliveira43) [reply samples/todolist_commit_deploy.md:1]: I did not read it carefully, but I am pretty sure this is a duplication of https://pelorus.readthedocs.io/en/latest/GettingStarted/QuickstartTutorial/ (and if so, no need to keep it)

Comment (mpryc) [reply samples/todolist_commit_deploy.md:1]: Yes this one is covered by QuickStart Tutorial.

**Review by mpryc — APPROVED** [review]

**Pull request merged**